demo/basic.html

```diff
@@ -0,0 +1,149 @@
+<!doctype html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+		<title>Cosmoz Grouped List</title>
+
+		<script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+		<link rel="import" href="../cosmoz-grouped-list.html">
+		<link rel="import" href="../../iron-flex-layout/iron-flex-layout.html"/>
+		<link rel="import" href="../../iron-flex-layout/iron-flex-layout-classes.html"/>
+		<link rel="import" href="../../iron-icons/iron-icons.html"/>
+		<link rel="import" href="../../iron-icon/iron-icon.html"/>
+
+		<style is="custom-style" include="iron-flex iron-flex-alignment iron-positioning">
+
+		</style>
+	</head>
+	<body class="fullbleed">
+		<dom-module id="demo-page">
+			<template>
+				<style include="iron-flex iron-flex-alignment iron-positioning">
+					.group-header {
+						padding: 5px;
+						background-color: #ccc;
+						font-weight: bold;
+					}
+
+					.extra-content.expanded {
+						display: block;
+					}
+
+					.extra-content {
+						display: none;
+					}
+
+					.action {
+						padding: 5px;
+						margin: 5px 5px 10px 5px;
+					}
+
+					.item-template {
+						@apply --layout-vertical;
+					}
+
+					.group-template {
+						@apply --layout-horizontal;
+						@apply --layout-center;
+						padding: 5px;
+						background-color: #ccc;
+						font-weight: bold;
+					}
+				</style>
+				<div class="layout vertical fit">
+					<h3>cosmoz-grouped-list demo</h3>
+					<cosmoz-grouped-list id="groupedList" data="{{ data }}" selected-items="{{ selectedItems }}" class="flex">
+						<cosmoz-grouped-list-template id="itemTemplate">
+							<template>
+								<div class="item-template" style="border-bottom: 1px solid grey;">
+									<div>
+										ID:<span class="item-id">{{ item.id }}</span>
+										NAME:<span class="item-name">{{ item.name }}</span>
+										VALUE:<span class="item-value">{{ item.value }}</span>
+										<div>
+								</div>
+							</template>
+						</cosmoz-grouped-list-template>
+						<cosmoz-grouped-list-template id="groupTemplate">
+							<template>
+								<div class="group-template">
+									NAME:<span class="item-name">{{ item.name }}</span>
+									VALUE: <span class="item-value">{{ item.value }}</span>
+									SELECTED: <span class="item-selected">{{ selected }}</span>
+								<div>
+							</template>
+						</cosmoz-grouped-list-template>
+					</cosmoz-grouped-list>
+				</div>
+
+			</template>
+		</dom-module>
+
+		<demo-page class="fit"></demo-page>
+
+		<script type="text/javascript">
+			(function () {
+				'use strict';
+
+				Polymer({
+					is: 'demo-page',
+					properties: {
+						data: {
+							type: Array,
+							notify: true,
+						},
+						selectedItems: {
+							type: Array
+						},
+					},
+
+					attached: function () {
+						this.async(function () {
+							var items = [
```

Comment:
This object structure is too spaced out, should be along the lines of:

```js
items: [{
  name: 'name',
  id: 'id'
}, {
  name: 'name2'
  id: 'id2'
}]
```

Also `items` isn't needed.

Any reason why this isn't just set on the `data` property's `value: function () { return.. }` ?